### PR TITLE
Add laundry to mem input plugin on FreeBSD

### PR DESCRIPTION
**Pull request by danielnelson**
closes: #7726

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/mem/memory.go [hunk 1]
```diff
@@ -61,6 +62,11 @@ func (s *MemStats) Gather(acc telegraf.Accumulator) error {
 		"write_back":        vm.Writeback,
 		"write_back_tmp":    vm.WritebackTmp,
 	}
+
+	if runtime.GOOS == "freebsd" {
+		fields["laundry"] = vm.Laundry
```
Comment (reimda) [inline plugins/inputs/mem/memory.go:67]: Need to add this to the docs, maybe with a note that it's specific to freebsd

Comment (reimda) [reply plugins/inputs/mem/memory.go:67]: We already support active, inactive, and wired which are only valid on OS X and BSD.  They're not handled conditionally at runtime.  Maybe this doesn't need to be either.

Comment (danielnelson) [reply plugins/inputs/mem/memory.go:54]: Active/inactive are set on Linux, despite the comment in gopsutil.  We really shouldn't be adding fields that aren't set, so I went through the stats again in more detail.  One situation I'm not handling yet is the non-cgo version of Darwin.

**Comment by danielnelson** [comment]
Now that we are returning different fields depending on the platform, we are running into the issue where we need the test to be different depending on the host running the tests.  To get around this I added a field to the struct that can be overridden in the tests to change the platform.

**Pull request merged**